raw/mock_exam/docs/questions/question-37.adoc

```diff
@@ -0,0 +1,92 @@
+// tag::DE[]
+=== Frage 37
+**ID: Q-20-04-28**
+
+[cols="2,8,2", frame=ends, grid=rows]
+|===
+| P-Frage: 
+| Wählen Sie die **vier** am besten passenden Antworten aus
+| 1 Punkt
+|===
+
+Welche vier der folgenden Punkte sind am besten geeignet, eine qualitative Analyse Ihrer Softwarearchitektur zu unterstützen?
+
+[cols="1a,1,10", frame=none, grid=none]
+|===
+
+| {y}
+| (a)
+| Quantitative Abhängigkeitsanalyse.
```

Comment:
Finde ich als Antwortmöglichkeit schwierig, weil die quantitative Analyse die qualitative Analyse unterstützen (im Sinne von "ergänzen") kann, aber die Fragestellung hier rein auf die Unterstützung der "qualitative Analyse einer Softwarearchitektur" abzielt.

Reply:
the option "quantative dependency analysis" imho definitely supports qualitative analysis (e.g. concerning the qualitities flexibility, understandability, maintainability...)